frontend/src/app/shared/Coordinates.ts

```diff
@@ -0,0 +1,4 @@
+export interface Coordinates {
+    latitude: number;
+    longitude: number;
+}
```

Comment:
Restaurant has already lat & lng ; maybe one day we can use this interface

Reply:
Yes it's a good idea